src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBadge/InfoBadge.properties.cs

```diff
@@ -9,43 +9,61 @@
 using System.Threading.Tasks;
 using Windows.UI.Xaml;
 
-namespace Microsoft.UI.Xaml.Controls
+namespace Microsoft.UI.Xaml.Controls;
+
+public partial class InfoBadge
 {
-	public partial class InfoBadge
+	/// <summary>
+	/// Gets or sets the integer to be displayed in a numeric InfoBadge.
+	/// </summary>
+	public int Value
+	{
+		get => (int)GetValue(ValueProperty);
+		set => SetValue(ValueProperty, value);
+	}
+
+	/// <summary>
+	/// Identifies the InfoBadge.Value dependency property.
+	/// </summary>
+	public static DependencyProperty ValueProperty { get; } =
+		DependencyProperty.Register(nameof(Value), typeof(int), typeof(InfoBadge), new FrameworkPropertyMetadata(-1, OnPropertyChanged));
+
+	/// <summary>
+	/// Provides calculated values that can be referenced as TemplatedParent sources when defining 
+	/// templates for an InfoBadge. Not intended for general use.
+	/// </summary>
+	public InfoBadgeTemplateSettings TemplateSettings
+	{
+		get => (InfoBadgeTemplateSettings)GetValue(TemplateSettingsProperty);
+		private set => SetValue(TemplateSettingsProperty, value);
```

Comment:
This will be a breaking change

Reply:
Yes, but safe one, nobody should be setting the property from the outside. Interestingly though - it did not cause diffignore error in CI 👀

Reply:
It's a safe one I agree, no problem adding it.